docs/setup.md

```diff
@@ -84,8 +84,7 @@ of directories shared in Docker.
       ```sh
        mkdir /oli/service_config
        cp /oli/sources/authoring-server/conf/content-service-conf.example.json /oli/service_config/content-service-conf.json
-       # make configuration modifications
-       vim /oli/service_config/content-service-conf.json
+       # make configuration modifications. A complete version of the file is located at https://github.com/Simon-Initiative/dev-docs/blob/master/content-service-conf.json
```

Comment:
dev-docs is not a public repo. We provide a sample config file as part of the authoring-server under conf https://github.com/Simon-Initiative/authoring-server/blob/master/conf/content-service-conf.example.json

Our instructions should include a step to copy this file to conf/content-service-conf.json if they dont already